docs/adding-zones.md

```diff
@@ -1,15 +1,12 @@
-# Adding Zones to a MinIO Cluster
+# Adding Zones to a MinIO Cluster [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-[![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
-[![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
-
-This document explains how to add zones to an existing MinIO Cluster with Operator. This document is only applicable to a MinIO Cluster created by MinIO Operator.
+This document explains how to add zones to an existing MinIO Cluster with Operator. This is only applicable to a MinIO Cluster created by MinIO Operator.
 
 Read more about MinIO Zones design in [MinIO Docs](https://github.com/minio/minio/blob/master/docs/distributed).
 
 ## Getting Started
 
-Assuming you have a MinIO cluster with single zone, `zone-0` with 4 drives (as shown in [examples](https://github.com/minio/operator/tree/master/examples)). You can dd a new zone `zone-1` with 4 drives using `kubectl patch` command.
+Assuming you have a MinIO cluster with single zone, `zone-0` with 4 drives (as shown in [examples](https://github.com/minio/operator/tree/master/examples)). You can add a new zone `zone-1` with 4 drives using `kubectl patch` command.
```

Comment:
So in the initial README quickstart we give the user a starting point. Does that starting point consist of a single 'zone'? IF so we can link to the quickstart to let users know that the example there creates the required cluster compatible with the given example.

Reply:
perhaps we should says `assuming you have a MinIO Tenant with a single zone`